code/game/machinery/cryopod.dm

```diff
@@ -335,7 +339,7 @@
 
 		var/preserve = null
 		for(var/T in preserve_items)
-			if(istype(W,T))
+			if(istype(W,T) && !(W in do_not_preserve_items))
```

Comment:
I'm open to suggestions - I think it's a huge edge case that someone will try to jump into a cryodorm just to delete an active posibrain, so I doubt that'll be a huge issue.